models/User.js

```diff
@@ -11,6 +11,7 @@ const User = mongoose.Schema({
                 scheduleIndices: [Number],
             },
         ],
+        scheduleNames: {type: [String], default: ['Schedule 1', 'Schedule 2', 'Schedule 3', 'Schedule 4']},
```

Comment:
By default, the frontend starts with on `['Schedule 1']`. Should this match that?